crates/superposition/src/main.rs

```diff
@@ -245,6 +245,11 @@ async fn main() -> Result<()> {
                             .wrap(AppExecutionScopeMiddlewareFactory::new(AppScope::CAC))
                             .service(type_templates::endpoints()),
                     )
+                    .service(
+                        scope("/config-versions")
```

Comment:
we already have `/config`, should this be `/config/versions`?

Reply:
@Datron thats a call to be made, if we want that then better move this handler to config handlers.